src/Domain/DonationRepository.php

```diff
@@ -107,6 +107,31 @@ public function buildFromApiRequest(DonationCreate $donationData): Donation
         $donation->setCharityComms($donationData->optInCharityEmail);
         $donation->setTbgComms($donationData->optInTbgEmail);
 
+        // Optionally set these donation fields if available
+        if (!empty($donationData->billingPostalAddress)) {
```

Comment:
@NoelLH I've just done an optional check for billingPostalAddress for now, we can revisit once we start sending the separate GA claim address